ReactiveCocoa/CocoaTarget.swift

```diff
@@ -4,16 +4,51 @@ import enum Result.NoError
 
 /// A target that accepts action messages.
 internal final class CocoaTarget<Value>: NSObject {
+	private enum State {
+		case idle
+		case sending(queue: [Value])
+	}
+
 	private let observer: Observer<Value, NoError>
 	private let transform: (Any?) -> Value
```

Comment:
It looks like this `transform` could be eliminated. It's only ever force-casted to `Value`.